Call member#to_h when writing to DB

ondenman opened the pull request:
The scraper is currently failing when writing to the database because it is passing Member objects instead of hashes into `ScraperWiki.save_sqlite`

Members holds a list of Member objects in its :members_of_the_lower_house
field. #to_h needs to be called on these objects when writing to the
database.

ondenman committed:
Call member#to_h when writing to DB

Members holds a list of Member objects in its :members_of_the_lower_house
field. #to_h needs to be called on these objects when writing to the
database.

tmtmtmtm commented on scraper.rb:
```diff
@@ -10,5 +10,5 @@
 members = Members.new(members_url).to_h[:members_of_the_lower_house]
 
 members.map do |member|
```
I realise it's a pre-existing condition, but rather than calling `map` in a void context like this, this should probably also be `.each`, and as the thing you're changing is the only thing that happens inside that loop, I think fixing it here makes sense.

ondenman committed:
Loop members with #each instead of #map

We're not assigning the resultant array to anything just performing
an operation using each item in the loop.
